package.json

```diff
@@ -48,11 +49,16 @@
     "eslint-plugin-no-only-tests": "^2.4.0",
     "eslint-plugin-prettier": "^3.1.2",
     "jest": "^25.1.0",
-    "prettier": "^1.19.1"
+    "prettier": "^1.19.1",
+    "rollup": "^1.30.1",
+    "rollup-plugin-commonjs": "^10.1.0",
+    "rollup-plugin-node-resolve": "^5.2.0"
   },
   "homepage": "https://github.com/mljs/cross-validation#readme",
   "dependencies": {
+    "jest-matcher-deep-close-to": "^1.3.0",
     "ml-combinations": "^1.1.0",
-    "ml-confusion-matrix": "^0.4.0"
+    "ml-confusion-matrix": "^0.4.0",
+    "ml-dataset-iris": "^1.1.1"
```

Comment:
this should be in devDependencies

Reply:
I moved data iris, the confusion matrix should be used by the prediction method, although it is not yet supported.